src/tests/board.test.ts

```diff
@@ -32,3 +32,25 @@ test("given two neighboring hexes we can get the direction from A to B", () => {
   const direction = board.findDirection([0, 0, 0], [0, 1, -1]);
   expect(direction).toBe(Direction.SE);
 });
+
+test("given an outer tile and a neighboring inner tile I can get a row of only the inner tiles", () => {
+  const row = board.getRow([4, 0], [3, 0]);
+  console.log("row length", row.size);
+  expect(row.filter((tile) => !tile.isOuterTile()).size).toEqual(row?.size);
+});
+
+test("given an outer tile not at the corner I can get a row of only the inner tiles", () => {
+  const row = board.getRow([4, -1], [3, 0]);
+  console.log("row length", row.size);
+  expect(row.filter((tile) => !tile.isOuterTile()).size).toEqual(row?.size);
```

Comment:
perhaps more readable to define expected and observed first, e.g. on a line expectedRowLength = row.size and then use within toEqual, as an example

Reply:
it is done